.gitlab-ci.yml

```diff
@@ -149,14 +132,7 @@ buildtest:search-no-index-cache:
   stage: buildtest
   image: registry.gitlab.com/magda-data/magda/data61/magda-builder-scala:$CI_COMMIT_REF_SLUG
   retry: 2
-  dependencies:
-    - builders-and-yarn
-    - sbt-prebuild
```

Comment:
We probably still want to pass build result via `artifacts`? Or I missed anything?